code/game/area/Space_Station_13_areas.dm

```diff
@@ -564,7 +562,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/engine/supermatter
 	name = "Supermatter Engine"
 	icon_state = "engine_sm"
-	valid_territory = FALSE
+	area_flags = BLOBS_ALLOWED | UNIQUE_AREA
```

Comment:
this can result in blobs dying instantly when they spawn since this is INSIDE the SM chamber.

Reply:
Err yes and no, blobs already do this and its considered "funny" to spawn in and smash the crystal 30 times in order to cause rad waves so that'd have to be a balance change